lib/cli.js

```diff
@@ -161,9 +178,15 @@ function usage() {
         '    grr TOOLS-123    # Start or update a branch and CR for this issue',
         //'    grr --commit TOOLS-123    # ... also commit uncommited changes',
         '    grr              # Update the current CR (using the cached issue)',
-	'    grr -a TOOLS-124 # Add an extra issue to the current CR',
-	'    grr -r TOOLS-124 # Remove an extra issue from the current CR',
-        '    grr -D           # All done. Del the branch and switch back to master'
+	    '    grr -a TOOLS-124 # Add an extra issue to the current CR',
+	    '    grr -r TOOLS-124 # Remove an extra issue from the current CR',
+    '    grr -D           # All done. Del the branch and switch back to master',
+        '',
+    '    grr -R cr TOOLS-124       # show status of reviews matching this issue',
```

Comment:
Yep, though it wasn't even successful at that, since line 185 has a comma dangling off the edge. I'll rework the examples section into two parts, "working on changes" and "reviewing changes" and see if we can squeeze in under 80 cols.